protocol/x/clob/keeper/liquidations.go

```diff
@@ -60,7 +72,7 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 	// Attempt to place each liquidation order and perform deleveraging if necessary.
 	numFilledLiquidations := uint32(0)
 	numAttemptedDeleveraging := uint32(0)
-	for i := 0; numFilledLiquidations < k.MaxLiquidationOrdersPerBlock && i < len(liquidationOrdersForSorting); i++ {
+	for i := 0; i < len(liquidationOrdersForSorting); i++ {
```

Comment:
Is it worth measuring the latency of how long it takes to liquidate subaccounts as well (will help determine how to configure the max liq orders per block)?

Reply:
going to do metrics in a separate PR